static/panes/diff.js

```diff
@@ -102,6 +103,7 @@ function Diff(hub, container, state) {
     this.domRoot.html($('#diff').html());
     this.compilers = {};
     var root = this.domRoot.find('.monaco-placeholder');
+    var self = this;
```

Comment:
Another self thing here, there's a few. I didn't tag them all in the last PR as I figured you might disagree and I didn't want to be _too_ annoying... thanks.